src/System.Device.Gpio/Interop/Unix/SafeLineHandle.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Text;
```

Comment:
Do we need all these using directives (particularly System.Text)?

Reply:
we havent particualrly paid attention to that in this repo. I'd presume we will eventually clean them all up in one sweep

Reply:
Seems like unnecessary tech debt, but ok :)

Reply:
removed